styles.css

```diff
@@ -1,312 +1,326 @@
-@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap");

```

Comment:
I can't see the real differences well. All the file looks modified. I thinks its my fault because I don't have the prettier enabled with css.

I added few things with 0.2.1  release. padding but I will merge and add again =) thanks!

Reply:
Oh, oops! VScode didn't show me any of these changes when I was committing.